sci-physics/vmc/vmc-1.0_p2.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+if [[ ${PV} == *9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/vmc-project/${PN}.git"
+else
+	MY_PV=$(ver_rs 1-2 -)
+	SRC_URI="https://github.com/vmc-project/${PN}/archive/v${MY_PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+	S="${WORKDIR}/${PN}-${MY_PV}"
+fi
+
+DESCRIPTION="The Virtual Monte Carlo core library."
+HOMEPAGE="https://vmc-project.github.io/"
+
+LICENSE="GPL-3"
+SLOT="0"
+IUSE="+c++11 c++14 c++17 doc"
+
+REQUIRED_USE="^^ ( c++11 c++14 c++17 )"
+
+RDEPEND="c++11? (
+		>=sci-physics/root-6.18:=[c++11,-vmc]
+	)
+	c++14? (
+		>=sci-physics/root-6.18:=[c++14,-vmc]
+	)
+	c++17? (
+		>=sci-physics/root-6.18:=[c++17,-vmc]
+	)"
+DEPEND="${RDEPEND}
+	doc? ( app-doc/doxygen[dot] )"
+
+DOCS=(README.md History)
+
+src_compile() {
+	cmake_src_compile
+	if use doc; then
+		TRAVIS_BUILD_DIR="${S}" doxygen doc/doxygen/Doxyfile || die
```

Comment:
Why setting `TRAVIS_BUILD_DIR` is necessary here?

Reply:
That's sadly an upstream thing:
https://github.com/vmc-project/vmc/blob/70e1c10855b40ac2c498f246747eb0f1eb8fec81/doc/doxygen/Doxyfile#L761-L763
i.e. they have hardcoded this environment variable name in their Doxyfile. 

Should I add a comment to the ebuild explaining the reasoning for future reference? 
Alternatively, I can also try to get this changed upstream, but I guess that they don't want to change their file structure (so probably it would just be a change of environment variable name).